qiskit/extensions/quantum_initializer/isometry.py

```diff
@@ -95,11 +96,7 @@ def __init__(self, isometry, num_ancillas_zero, num_ancillas_dirty):
         super().__init__("isometry", num_qubits, 0, [isometry])
 
     def _define(self):
-        # call to generate the circuit that takes the isometry to the first 2^m columns
-        # of the 2^n identity matrix
-        iso_circuit = self._gates_to_uncompute()
-        # invert the circuit to create the circuit implementing the isometry
-        gate = iso_circuit.to_instruction().inverse()
+        gate = self.inverse().inverse()
```

Comment:
It might be helpful to add a comment explaining.

Reply:
Something like 758560666 ?